faculty_cli/cli.py

```diff
@@ -1239,3 +1240,15 @@ def ls(project, path):
             click.echo("/project{}/".format(item.path))
         else:
             click.echo("/project{}".format(item.path))
+
+
+@cli.command()
+@click.argument("shell", type=click.Choice(["bash", "zsh", "fish"]))
+def completion(shell):
+    """Generate auto-completion scripts for faculty_cli."""
```

Comment:
Can we add a link to the usage information in the README? e.g.:

```python
"""Generate auto-completion scripts for faculty_cli.

See https://github.com/facultyai/faculty-cli/blob/master/README.md
for further info on blah blah blah.
"""
```

I believe click prints the first line of the docstrings when listing subcommands, and the full help when running `--help` on the subcommand itself.